### PR TITLE
!per #17862 Remove experimental from persistence

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

**Comment by patriknw** [comment]
Refs #17862

**Comment by drewhk** [comment]
LGTM

File: akka-docs/rst/project/migration-guide-2.3.x-2.4.x.rst [hunk 1]
```diff
@@ -300,6 +300,26 @@ For the synchronous case you can return the result via ``scala.concurrent.Future
...
+
+New Maven dependency::
+
+  <dependency>
+    <groupId>com.typesafe.akka</groupId>
+    <artifactId>akka-persistence_@binVersion@</artifactId>
+    <version>@version@</version>
+  </dependency>
+
+The artefact name of the Persistent TCK has changed from ``akka-persistence-tck-experimental`` (``akka-persistence-experimental-tck``) to
```
Comment (ktoso) [inline akka-docs/rst/project/migration-guide-2.3.x-2.4.x.rst]: artifact

Comment (patriknw) [inline akka-docs/rst/project/migration-guide-2.3.x-2.4.x.rst]: right, thx

Comment (ktoso) [inline akka-docs/rst/project/migration-guide-2.3.x-2.4.x.rst]: btw should this have `experimental` here still?

**Comment by ktoso** [comment]
LGTM, yay!

File: akka-docs/rst/project/migration-guide-2.3.x-2.4.x.rst [hunk 2]
```diff
@@ -300,6 +300,26 @@ For the synchronous case you can return the result via ``scala.concurrent.Future
...
+``akka-persistence-tck``.
+
 Mendatory persistenceId
```
Comment (ahjohannessen) [inline akka-docs/rst/project/migration-guide-2.3.x-2.4.x.rst]: I presume it should be *mandatory* ?

**Pull request merged**